### PR TITLE
Examples update

### PR DESCRIPTION
**Pull request by raviganapatp**

### CODE REVIEW

File: canary/README.md [hunk 1]
```diff
@@ -802,22 +802,25 @@ Perform the following steps to deploy a sample application as a canary release.
...
-                      backend:
-                        serviceName: guestbook-canary
-                        servicePort: 80
+                    - backend:
+                        service:
+                          name: guestbook-canary
+                          port:
+                            number: 80
+                      path: /
+                      pathType: ImplementationSpecific
```
Comment (kumar-swamy) [inline canary/README.md:823]: I'd suggest you to use pathType as Prefix rather than ImplementationSpecific in all example

Comment (raviganapatp) [reply canary/README.md]: Done

File: canary/manifest/cpx-exporter.yaml [hunk 2]
```diff
@@ -1,68 +1,66 @@
-apiVersion: extensions/v1beta1
```
Comment (aroraharsh23) [inline canary/manifest/cpx-exporter.yaml:1]: Can we make changes in all the exporter file as per the example here : https://github.com/citrix/citrix-k8s-ingress-controller/tree/master/metrics-visualizer#configuring-citrix-adc-metrics-exporter
Looks, like these canary and other examples were not updated. Let me know if case of any doubt.

File: docs/network/type_loadbalancer.md [hunk 3]
```diff
@@ -186,28 +186,35 @@ Perform the following steps to deploy the IPAM controller.
...
-                image: quay.io/citrix/citrix-ipam-controller:latest
+              - image: quay.io/citrix/citrix-ipam-controller:latest
+                imagePullPolicy: Always
+                name: citrix-ipam-controller
                 env:
                 # This IPAM controller takes environment variable VIP_RANGE. IPs in this range are used to assign values for IP range
-              - name: "VIP_RANGE"
-                value: '["10.105.158.195/32", "10.105.158.196/31", "10.105.158.198"]'
+              - imagePullPolicy: IfNotPresent
+                name: VIP_RANGE
```
Comment (apoorva-05) [inline docs/network/type_loadbalancer.md:213]: VIP range example shouldn't have been removed.

File: docs/metrics/promotheus-grafana.md [hunk 4]
```diff
@@ -144,63 +144,67 @@ To monitor a Citrix ADC CPX ingress device, the Citrix ADC metrics exporter is a
 
 ```YAML
 ---
-apiVersion: extensions/v1beta1
```
Comment (aroraharsh23) [inline docs/metrics/promotheus-grafana.md:147]: same as above, refer Can we make changes in all the exporter file as per the example here : https://github.com/citrix/citrix-k8s-ingress-controller/tree/master/metrics-visualizer#configuring-citrix-adc-metrics-exporter
Looks, like these canary and other examples were not updated. Let me know if case of any doubt.

File: docs/metrics/promotheus-grafana.md [hunk 5]
```diff
@@ -144,63 +144,67 @@ To monitor a Citrix ADC CPX ingress device, the Citrix ADC metrics exporter is a
...
+      # Adding exporter as a sidecar
+      - args:
+        - --target-nsip=192.0.0.2
+        - --port=8888
+        - --secure=no
+        env:
+        - name: NS_USER
+          value: nsroot
+        - name: NS_PASSWORD
+          value: nsroot
```
Comment (apoorva-05) [inline docs/metrics/promotheus-grafana.md:201]: If possible can the credentials be removed from this yaml

File: crd/ratelimit/README.md [hunk 6]
```diff
@@ -15,82 +15,97 @@ The Rate limit CRD is available in the Citrix ingress controller GitHub repo at:
 The following is the Rate limit CRD definition ([ratelimit-crd.yaml](https://raw.githubusercontent.com/citrix/citrix-k8s-ingress-controller/master/crd/ratelimit/ratelimit-crd.yaml)):
 
 ```yml
```
Comment (kumar-swamy) [inline crd/ratelimit/README.md:17]: This should not be part of the document, can we remove this from the document

File: canary/manifest/cpx-with-cic-sidecar.yml [hunk 7]
```diff
@@ -1,86 +1,88 @@
...
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+        - name: default
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        image: quay.io/citrix/citrix-k8s-ingress-controller:1.7.6
```
Comment (apoorva-05) [inline canary/manifest/cpx-with-cic-sidecar.yml:59]: can the CIC image be updated

**Review by apoorva-05 — CHANGES_REQUESTED** [review]

File: docs/configure/ingress-config.md [hunk 8]
```diff
@@ -66,20 +78,23 @@ Citrix ingress controller follows first match policy to evaluate paths. For effe
 The following sample Ingress definition demonstrates how to set up an ingress with wildcard host.
 
 ```yml
-apiVersion: extension/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: Wildcard-Ingress
```
Comment (apoorva-05) [inline docs/configure/ingress-config.md:84]: ingress name has to be in lowercase

File: docs/configure/ingress-config.md [hunk 9]
```diff
@@ -66,20 +78,23 @@ Citrix ingress controller follows first match policy to evaluate paths. For effe
...
   namespace: default
 spec:
   rules:
-  - host:’*.example.com’
+  - host: ’*.example.com’
```
Comment (apoorva-05) [inline docs/configure/ingress-config.md:88]: needs to be replaced with '*.example.com'

File: crd/bot/bot-crd.yaml [hunk 10]
```diff
@@ -161,4 +161,4 @@ spec:
                   type: array
                   properties:
                     urls:
-                      type: object
+                      type: object
```
Comment (kumar-swamy) [inline crd/bot/bot-crd.yaml:164]: Avoid this change as this is changed by Ganesh

File: docs/configure/ingress-config.md [hunk 11]
```diff
@@ -94,22 +109,25 @@ By default Ingress paths are treated as prefix expressions. Using the annotation
...
-        backend:
-          serviceName: service1
-          servicePort: '80'
+      - backend:
+          service:
+            name: service1
+            port:
+              name: "80"
+        path: /exact
+        pathType: Prefix
```
Comment (apoorva-05) [inline docs/configure/ingress-config.md:130]: Replace with the following Ingress.
```yml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    ingress.citrix.com/path-match-method: "exact"
  name: path-exact-ingress
  namespace: default
spec:
  rules:
  - host: test.example.com
    http:
      paths:
      - backend:
          service:
            name: service1
            port:
              number: 80
        path: /exact
        pathType: Exact

```

File: docs/configure/ingress-config.md [hunk 12]
```diff
@@ -119,23 +137,29 @@ After the sample Ingress definition is deployed, HTTP requests with path `/exact
 Following example shows path based routing for the default traffic that does not match any host based routes. This ingress rule applies to all inbound HTTP traffic through the specified IP address.
 
 ```yml
-apiVersion: extension/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: Default-Path-Ingress
```
Comment (apoorva-05) [inline docs/configure/ingress-config.md:143]: need to convert ingress name to lowercase

File: docs/configure/ingress-config.md [hunk 13]
```diff
@@ -145,15 +169,17 @@ All incoming traffic that does not match the ingress rules with host name is mat
 Default back end is a service that handles all traffic that is not matched against any of the Ingress rules.
 
 ```yml
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: Default-Ingress
```
Comment (apoorva-05) [inline docs/configure/ingress-config.md:175]: ingress name has to be converted to lowercase

File: docs/deploy/deploy-cic-openshift.md [hunk 14]
```diff
@@ -125,69 +125,72 @@ Perform the following steps to deploy Citrix ADC CPX as a router with the Citrix
...
-                        fieldPath: metadata.namespace
-                  args:
-                    - --default-ssl-certificate
-                      $(POD_NAMESPACE)/default-cert
-                  imagePullPolicy: Always
+              - env:
+                - name: EULA
+                  value: "yes"
+                - name: KUBERNETES_TASK_ID
+                image: quay.io/citrix/citrix-k8s-cpx-ingress:13.0-36.29
```
Comment (apoorva-05) [inline docs/deploy/deploy-cic-openshift.md:150]: RBAC has to be updated; 
CIC Image version has to be updated line 40 and line187 .

File: docs/deploy/deploy-cic-yaml.md [hunk 15]
```diff
@@ -203,7 +203,7 @@ rules:
 ---
 
 kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (apoorva-05) [inline docs/deploy/deploy-cic-yaml.md:206]: Update the RBAC

File: docs/how-to/MongoDB-single-tier-solution.md [hunk 16]
```diff
@@ -104,20 +104,22 @@ Perform the following steps to deploy the Citrix Ingress solution for MongoDB.
...
+            ingress.citrix.com/insecure-service-type: mongo
+            ingress.citrix.com/insecure-termination: allow
+            kubernetes.io/ingress.class: tier-1-vpx
+          name: vpx-ingress
+        spec:
+          defaultBackend:
+            service:
+              name: mongodb-mongos
+              port:
+                number: 27017
```
Comment (apoorva-05) [inline docs/how-to/MongoDB-single-tier-solution.md:123]: best to place inside yml code block
```yml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    ingress.citrix.com/analyticsprofile: '{"tcpinsight": {"tcpBurstReporting":"DISABLED"}}'
    ingress.citrix.com/insecure-port: "27017"
    ingress.citrix.com/insecure-service-type: mongo
    ingress.citrix.com/insecure-termination: allow
    kubernetes.io/ingress.class: tier-1-vpx
  name: vpx-ingress
spec:
  defaultBackend:
      service:
         name: mongodb-mongos
         port:
           number: 27017

```

File: docs/how-to/examples-yamls/cic-k8s-role-service-account.yml [hunk 17]
```diff
@@ -7,7 +7,7 @@ metadata:
 ---
 
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (apoorva-05) [inline docs/how-to/examples-yamls/cic-k8s-role-service-account.yml:10]: Update RBAC

File: docs/configure/ingress-config.md [hunk 18]
```diff
@@ -94,22 +109,25 @@ By default Ingress paths are treated as prefix expressions. Using the annotation
 The following sample Ingress definition demonstrates how to set up Ingress for exact path matching:
```
Comment (apoorva-05) [inline docs/configure/ingress-config.md:110]: Replace ``` By default Ingress paths are treated as prefix expressions. Using the annotation ingress.citrix.com/path-match-method: “exact” in the ingress definition defines the Citrix ingress controller to consider the path for the exact match." ``` with

File: docs/how-to/grpc.md [hunk 19]
```diff
@@ -141,52 +141,52 @@ Perform the following steps to enable GRPC support using HTTP2.
   
      The content of the `frontend-ingress.yaml` file is provided as follows:
 
-          apiVersion: extensions/v1beta1
+          apiVersion: networking.k8s.io/v1
           kind: Ingress
```
Comment (apoorva-05) [inline docs/how-to/grpc.md:145]: best to place inside yml code block 
```yml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
     ingress.citrix.com/frontend-httpprofile: '{"http2":"enabled", "http2direct" : "enabled"}'
     ingress.citrix.com/frontend-ip: 192.0.2.1
     ingress.citrix.com/secure-port: "443"
     kubernetes.io/ingress.class: citrix
  name: frontend-ingress
spec:
  rules:
  - {}
  tls:
  - {}

```

File: docs/how-to/grpc.md [hunk 20]
```diff
@@ -141,52 +141,52 @@ Perform the following steps to enable GRPC support using HTTP2.
...
+                    service:
+                      name: grpc-service
+                      port:
+                        number: 50051
+                  path: /
+                  pathType: Prefix
+            tls:
+            - hosts:
+              - grpc.example.com
+              secretName: grpc-secret
```
Comment (apoorva-05) [inline docs/how-to/grpc.md:190]: Best to place inside yml code block.
```yml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
   annotations:
       ingress.citrix.com/backend-httpprofile: '{"grpc-service":{"http2": "enabled", "http2direct" : "enabled"}}'
       ingress.citrix.com/frontend-ip: 192.0.2.2
       ingress.citrix.com/secure-port: "443"
       kubernetes.io/ingress.class: citrix
   name: grpc-ingress
spec:
  rules:
  - host: grpc.example.com
    http:
      paths:
      - backend:
           service:
             name: grpc-service
             port:
                number: 50051
        path: /
        pathType: Prefix
  tls:
  - hosts:
    - grpc.example.com
    secretName: grpc-secret

```

File: docs/how-to/hpa/ingress_vpx.yaml [hunk 21]
```diff
@@ -1,17 +1,19 @@
...
-      - path:
-        backend:
-          serviceName: cpx-builtin
-          servicePort: 6080
+      - backend:
+          service:
+            name: cpx-builtin
+            port:
+              number: 6080
+        pathType: Prefix
```
Comment (apoorva-05) [inline docs/how-to/hpa/ingress_vpx.yaml:19]: path: / has to be added

File: docs/how-to/hpa/rbac.yaml [hunk 22]
```diff
@@ -38,7 +38,7 @@ rules:
     verbs: ["get", "list", "watch"]
 ---
 kind: ClusterRoleBinding
```
Comment (apoorva-05) [inline docs/how-to/hpa/rbac.yaml:40]: Update RBAC

File: docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml [hunk 23]
```diff
@@ -335,39 +341,45 @@ spec:
...
     http:
       paths:
       - backend:
-          serviceName: frontend-headless
-          servicePort: 80
+          service:
+            name: frontend-headless
+            port:
+              number: 80
+        pathType: Prefix
```
Comment (apoorva-05) [inline docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml:362]: path: / has to be added.

File: docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml [hunk 24]
```diff
@@ -335,39 +341,45 @@ spec:
...
```
Comment (apoorva-05) [inline docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml:382]: path: / has to be added.

File: docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml [hunk 25]
```diff
@@ -394,22 +406,25 @@ spec:
...
     http:
       paths:
       - backend:
-          serviceName: frontend-external-headless
-          servicePort: 80
+          service:
+            name: frontend-external-headless
+            port:
+              number: 80
+        pathType: Prefix
```
Comment (apoorva-05) [inline docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml:427]: path: / has to be added.

File: docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml [hunk 26]
```diff
@@ -1272,7 +1304,7 @@ rules:
     verbs: ["get", "list", "watch"]
 ---
 kind: ClusterRoleBinding
```
Comment (apoorva-05) [inline docs/how-to/sml/example/hipster-smlite-all-in-one-dual-tier.yaml:1306]: RBAC has to be updated

File: docs/how-to/sml/example/hipster-smlite-all-in-one-unified-ingress.yaml [hunk 27]
```diff
@@ -1057,7 +1080,7 @@ rules:
     verbs: ["get", "list", "watch"]
```
Comment (apoorva-05) [inline docs/how-to/sml/example/hipster-smlite-all-in-one-unified-ingress.yaml:1080]: RBAC has to be updated

**Review by apoorva-05 — CHANGES_REQUESTED** [review]

File: example/dsr/KubernetesConfig/guestbook-ingress.yaml [hunk 28]
```diff
@@ -1,11 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: guestbook-ingress
   annotations:
-   kubernetes.io/ingress.class: "dsr-tier2"
-   ingress.citrix.com/frontend-ip: "10.102.169.210"
+    ingress.citrix.com/frontend-ip: 10.102.169.210
```
Comment (aroraharsh23) [inline example/dsr/KubernetesConfig/guestbook-ingress.yaml:5]: remove the IP

File: example/servicegraph-demo/team_hotdrink.yaml [hunk 29]
```diff
@@ -126,40 +126,44 @@ spec:
...
-        backend:
-          serviceName: frontend-hotdrinks
-          servicePort: 80
-  - host:  tea
+      - backend:
+          service:
+            name: frontend-hotdrinks
+            port:
+              number: 80
+        pathType: Prefix
```
Comment (aroraharsh23) [inline example/servicegraph-demo/team_hotdrink.yaml:146]: Add Path

File: example/servicegraph-demo/team_hotdrink.yaml [hunk 30]
```diff
@@ -126,40 +126,44 @@ spec:
...
-      - path: 
-        backend:
-          serviceName: tea-beverage
-          servicePort: 80
-  - host:  coffee
+      - backend:
+          service:
+            name: tea-beverage
+            port:
+              number: 80
```
Comment (aroraharsh23) [inline example/servicegraph-demo/team_hotdrink.yaml:154]: Add Path

File: example/servicegraph-demo/team_hotdrink.yaml [hunk 31]
```diff
@@ -126,40 +126,44 @@ spec:
...
-      - path: 
-        backend:
-          serviceName: coffee-beverage
-          servicePort: 443
+      - backend:
+          service:
+            name: coffee-beverage
+            port:
+              number: 443
+        pathType: Prefix
```
Comment (aroraharsh23) [inline example/servicegraph-demo/team_hotdrink.yaml:164]: Add Path

File: docs/network/nodeport.md [hunk 32]
```diff
@@ -83,14 +83,16 @@ The sample deploys and exposes the apache web server as a service. You can acces
 After you have deployed the service, create an ingress definition to configure the ingress Citrix ADC to send the North-South traffic to the nodeport of the `apache` app. The following is a manifest for the ingress definition:
```
Comment (aroraharsh23) [inline docs/network/nodeport.md:83]: please update line no.22

File: docs/network/type_loadbalancer.md [hunk 33]
```diff
@@ -186,28 +186,35 @@ Perform the following steps to deploy the IPAM controller.
...
                 # The IPAM controller can also be configured with name spaces for which it would work through the environment variable
                 # VIP_NAMESPACES, This expects a set of namespaces passed as space separated string
+              serviceAccountName: citrix-ipam-controller
+
 
     The manifest contains two environment variables, `VIP_RANGE` and `VIP_NAMESPACES`. You can specify the appropriate routable IP range with a valid CIDR under the `VIP_RANGE`. If necessary, you can also specify a set of namespaces under  `VIP_NAMESPACES`  so that the IPAM controller allocates addresses only for services from specific namespaces.
```
Comment (aroraharsh23) [inline docs/network/type_loadbalancer.md:220]: Please check line 236

File: multicluster/Manifest/gtp-crd.yaml [hunk 34]
```diff
@@ -36,106 +16,134 @@ spec:
...
+            type: object
+          status:
+            type: object
+            properties:
+              state:
+                 type: string
+              status_message:
+                 type: string
+          spec:
+            required: ["serviceType","hosts","status"]
```
Comment (kumar-swamy) [inline multicluster/Manifest/gtp-crd.yaml:60]: status is not required field here, please take the correct spec for this CRD from viswa.

File: docs/multicluster/multi-cluster.md [hunk 35]
```diff
@@ -355,109 +335,138 @@ spec:
...
+            type: object
+          status:
+            type: object
+            properties:
+              state:
+                 type: string
+              status_message:
+                 type: string
+          spec:
+            required: ["serviceType","hosts","status"]
```
Comment (kumar-swamy) [inline docs/multicluster/multi-cluster.md:379]: "status" is not required field, take the correct spec from Viswa.

File: docs/multicluster/multi-cluster.md [hunk 36]
```diff
@@ -317,34 +317,14 @@ The following is the GTP CRD definition.
 
 ```yml
```
Comment (kumar-swamy) [inline docs/multicluster/multi-cluster.md:318]: This YAML should not be part of the documentation, it needs to be link to a file

File: docs/certificate-management/ssl-passthrough.md [hunk 37]
```diff
@@ -23,25 +23,28 @@ SSL passthrough is enabled for all services or host names provided in the Ingres
...
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: hotdrinks-ingress
   annotations:
-    ingress.citrix.com/frontend-ip: "10.106.143.160" # IP address of the Ingress Citrix ADC.
-    kubernetes.io/ingress.class: "citrix"
-    ingress.citrix.com/ssl-passthrough: "True"  # Enable SSL passthrough on the Ingress Citrix ADC
-    ingress.citrix.com/insecure-termination: "redirect"
+    ingress.citrix.com/frontend-ip: 10.106.143.160
```
Comment (kumar-swamy) [inline docs/certificate-management/ssl-passthrough.md:30]: Chnage the internal IP

**Review by viswanath-ys — CHANGES_REQUESTED** [review]
Update crd/auth/auth_example1.yaml as mentioned below...

apiVersion: citrix.com/v1beta1
kind: authpolicy
metadata:
  name: authexample
spec:
    servicenames:
    - frontend

    authentication_providers:

        - name: "local-auth-provider"
          basic_local_db:
             use_local_auth: 'YES'

    authentication_policies:

        - resource:
            path:
              - '/orders/'
              - '/shipping/'
            method: [GET, POST]
          provider: ["local-auth-provider"]

        - resource:
            path:
              - '/products/'
            method: [POST]
          provider: ["local-auth-provider"]

          # no auth for these
        - resource:
            path:
              - '/products/'
            method: [GET]
          provider: []

        - resource:
            path:
              -  '/reviews/'
          provider: []



Update docs/deploy/deploy-api-gateway-using-gitops.md as mentioned in the mail...

**Review by viswanath-ys — CHANGES_REQUESTED** [review]
Update docs/deploy/deploy-api-gateway-using-gitops.md as mentioned in the mail to Sreejith...

1.	Update the description for oas_secret_ref as mentioned below
Specifies the Git access secret reference as a Kubernetes secret object name. Note: When creating a secret, keep the username and password as the secret field names for git access credentials.

2.	Remove the 2 fields mentioned below (these are not in use)
User_name	Specifies the Git user name.
Password	Specifies the Git password. Note: Credentials can be specified as a git_secret_ref as mentioned before or as user name and password in plain text format.

3) 3.	Update the example as mentioned below.
apiVersion: citrix.com/v1beta1
kind: apigatewaypolicy
metadata:
  name: apigatewaypolicyinstance
spec:
    api_definition:
        repository: "https://code.citrite.net/scm/cnn/cic-gitops.git"
        branch: "modify-test-branch"
        oas_secret_ref: "mysecret"
        #user_name: "vi"
        #password: "vi"
        files:
            - "test_gitwatcher/petstore.yaml"
            - "test_gitwatcher/playstore.yaml"
 
    api_proxy:
        ipaddress: "10.106.172.83"
        port:  80
        protocol: "http"
        secret: "listner-secret"
 
    policies:
       - name: "p1"
         selector:
            - api: "/pet.*"
              method: ["GET", "POST"] 
         upstream:
            service: "pet-service"
            port: 80
         policy_bindings:
            ratelimit:
                name: "ratelimit-gitops-slow"
 
       - name: "p2"
         selector:
            - api: "/user.*"
              method: ["GET", "POST"] 
         upstream:
            service: "user-service"
            port: 80
         policy_bindings:
            ratelimit:
                name: "ratelimit-gitops-slow"
 
       - name: "p3"
         selector:
            - tags: ["play"]
         upstream:
            service: "play-service"
            port: 80
         policy_bindings:
            ratelimit:
                name: "ratelimit-gitops"
            rewritepolicy:
                name: "prefixurl"
            waf:
                name: "buffoverflow"
    aaa:
       - crd_name: authgitops
         mappings:
           - petstore_auth: jwt-auth-provider
           - api_key: introspect-auth-provider

File: canary/manifest/spin-ingress-ssl.yaml [hunk 38]
```diff
@@ -1,34 +1,38 @@
...
-      - spinnaker-api.example.com
-      secretName: newcert
   rules:
   - host: spinnaker.example.com
     http:
       paths:
       - backend:
-          serviceName: spin-deck
-          servicePort: 9000
-        path:
```
Comment (bharthi-m) [inline canary/manifest/spin-ingress-ssl.yaml:27]: path is missing in the updated ingress.

File: canary/manifest/spin-ingress-ssl.yaml [hunk 39]
```diff
@@ -1,34 +1,38 @@
...
+            port:
+              number: 9000
+        pathType: Prefix
   - host: spinnaker-api.example.com
     http:
       paths:
       - backend:
-          serviceName: spin-gate
-          servicePort: 8084
-        path:
```
Comment (bharthi-m) [inline canary/manifest/spin-ingress-ssl.yaml:34]: path is missing in the updated ingress.

**Pull request merged**